app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt

```diff
@@ -107,8 +112,6 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
     override fun goToMetadata() {
         val intent = Intent(this, MetadataActivity::class.java)
         startActivity(intent)
-        //Finish the activity to avoid keeping all the bitmaps in memory
-        finish()
```

Comment:
Are you sure we should remove this? I am not sure how changing the bindings fixes the issue we had with keeping the Activities (and thus the Bitmaps) on the stack.

Also, when I take a heapdump I see multiple MaskActivities again

Reply:
👍